docs/platform/visual/index.md

```diff
@@ -11,10 +11,9 @@ The Pentaho Visualization API provides a unified way to visualize data across th
 [Analyzer](http://www.pentaho.com/product/business-visualization-analytics), 
 [PDI](http://www.pentaho.com/product/data-integration), 
 [CDF](http://community.pentaho.com/ctools/cdf/)).
-Essentially, it is a set of abstractions that ensures total isolation between 
-applications, 
-visualizations and 
-configurations (the glue).
+
+Essentially, it is a set of abstractions that ensures isolation between
+applications, visualizations and configurations.
```

Comment:
I think the glue metaphor isn't that clear. We understand it because that's what we've been calling it. Maybe "that glue the two together" or just remove it.